src/components/GestureTilePart.svelte

```diff
@@ -12,6 +12,8 @@
   export let selected = false;
 </script>
 
+<!-- svelte-ignore a11y-no-static-element-interactions -->
+<!-- Not a button as it would cause nesting of buttons -->
```

Comment:
I don't think we should suppress this warning as it could well be a legitimate problem and I'd like to leave it flagged.

Reply:
Ok!